### PR TITLE
Add InterNetX

### PR DESCRIPTION
**Pull request by InterNetX**
Add InterNetX Icon and Data

### CODE REVIEW

**Review by stephengroat — CHANGES_REQUESTED** [review]
@InterNetX we tend to only list companies under a single category. what do you think the primary category for InterNetX is, maybe `domains`?

File: _data/domains.yml [hunk 1]
```diff
@@ -262,6 +262,14 @@ websites:
       software: Yes
       doc: https://www.inwx.de/en/offer/mobiletan
 
+    - name: InterNetX
+      url: https://www.internetx.com
+      img: internetx.png
+      tfa: Yes
+      sms: Yes
+      software: Yes
+      doc: https://www.internetx.com/en/domains/autodns/
```
Comment (Jawshy) [inline _data/domains.yml:271]: I cannot see any documentation about SMS or software 2FA on that webpage. Are you sure that is the correct page?

**Review by Jawshy — CHANGES_REQUESTED** [review]
`doc`

File: _data/domains.yml (hunk 1 above)
Comment (InterNetX) [reply _data/domains.yml:271]: There it is https://www.internetx.com/en/domains/autodns/domain-safe/ or do you need full documentation?

Comment (Jawshy) [reply _data/domains.yml:271]: Full documentation would be ideal.

**Review by stephengroat — APPROVED** [review]

**Comment by Carlgo11** [comment]
How should we continue with this PR?

**Comment by Jawshy** [comment]
Full documentation or close.

**Comment by Carlgo11** [comment]
And since the last response from the author is on the 21st of November I guess it's the later option.

**Pull request closed without merging**